blocked-edges/4.17.0-OAuthServerDownIfSpaceInIDPName.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.17.0
+from: .*
```

Comment:
should be `from: 4[.]16[.].*`, because 4.17 is already exposed, so 4.17 -> 4.17 doesn't make anything worse?

Reply:
Got it.